libs/sound/src/PortAudioSound.cpp

```diff
@@ -147,12 +165,13 @@ static bool writeSynchronous(int deviceIx,
         }
     }
     Pa_CloseStream(stream);
+    free(buff);
     return pe == paNoError;
 }
 
 PortAudioSound::PortAudioSound()
     : m_soundLoader(SoundLoaderFactory()),
-    m_lock(ATOMIC_FLAG_INIT)
+    m_lock()
```

Comment:
Why? 
https://en.cppreference.com/w/cpp/atomic/atomic_flag/atomic_flag
`Trivial default constructor, initializes std::atomic_flag to unspecified state.`

Reply:
I will change in in class definition to   m_lock = ATOMIC_FLAG_INIT;